frontend/common/services/useFeatureList.ts

```diff
@@ -0,0 +1,148 @@
+import { Res } from 'common/types/responses'
+import { Req } from 'common/types/requests'
+import { service } from 'common/service'
+import { config } from 'common/config'
+
+export const featureListService = service
+  .enhanceEndpoints({
+    addTagTypes: [
+      'FeatureList',
+      'FeatureState',
+      'MultivariateOption',
+      'Environment',
+    ],
+  })
+  .injectEndpoints({
+    endpoints: (builder) => ({
+      createMultivariateOption: builder.mutation<
```

Comment:
just to confirm. This one and the delete operation have been created but not used right ?